src/Entities/Misc/RailedMoveBlock.cs

```diff
@@ -254,6 +260,15 @@ public override void Awake(Scene scene)
 
         scene.Add(border = new Border(this));
         scene.Add(pathRenderer = new RailedMoveBlockPathRenderer(this));
+
+
+        if (attachedAbove)
+        {
+            foreach (StaticMover staticMover in this.staticMovers)
+            {
+                staticMover.Entity.Depth = this.Depth - 1;
+            }
```

Comment:
```suggestion
            foreach (StaticMover staticMover in staticMovers)
            {
                staticMover.Entity.Depth = Depth - 1;
            }
```
Tiny nit, `this` is redundent here

Reply:
fixed